pyproject.toml

```diff
@@ -2,7 +2,7 @@
 # See LICENSE file for licensing details.
 
 [tool.poetry]
-name = "postgesql-k8s"
+name = "postgresql-k8s"
```

Comment:
o_0 how it worked?

Reply:
This is just poetry metadata, it requires it but it's not really used for anything.